.github/workflows/new_theme_bundle.yml

```diff
@@ -0,0 +1,39 @@
+name: New theme bundle
+
+on:
+  repository_dispatch:
```

Comment:
❓ Can we also addd a workflow_dispatch for manual sending?
We have something similar in `publish-production.yml`. This requires some minor changes in the `env` configuration.